### DISCUSSION TITLE
WiiMote orientation issue

### DISCUSSION THREAD

**Discussion by NZedPred** [v1993/linuxmotehook2#6] [Q&A]
I have managed to get this working with Cemu. An issue I have found is that the standard accelerometer controls are misaligned. For example, in Mario Kart 8, holding the Wiimote horizontally, I have to rotate it on its side (so that the top buttons are facing towards me), so that I can steer left/right correctly.

This is my config:

[Linuxmotehook]
Port=26760
Orientation=sideways-left
SendButtons=false
GyroNormalizationFactor=50

[0x00191d781df0]
GyroCalibration=0,0,0
Orientation=normal
SendButtons=true
NunchuckStickCalibration=0,2,95,83
ClassicControllerStickCalibration=0,0,25,25,0,0,25,25
ProControllerStickCalibration=0,0,1000,1000,0,0,1000,1000`

**Comment by v1993** [comment]
Can't reproduce on my end - I don't have that specific game to test tho. What input device are you using in Cemu? Also, maybe a stupid question, but - are you sure this isn't how you're *supposed* to hold wiimote in this game?
